apps/yapms/src/routes/app/+layout.svelte

```diff
@@ -29,7 +29,7 @@
 			LoadingErrorModalStore.set({
 				open: true,
 			});
-			await goto(`/app/usa/presidential/2022`);
+			await goto($page.url);
```

Comment:
This can go back to redirecting to ```await goto("/app/usa/presidential/2022");``` because it should be the default if something goes wrong.